crates/services/operatorsinfo/src/operatorsinfo_inmemory.rs

```diff
@@ -260,8 +327,17 @@ impl OperatorInfoServiceInMemory {
             tokio::select! {
                 _ = cancellation_token.cancelled() => {
                     self.logger.info("Cancellation signal received, stopping the stream.", "eigen-services-operatorsinfo.start_service");
+                    handle.abort();
                     break;
                 },
+                res = rx.recv() => {
+                    match res {
+                        Some(Err(err)) => {
```

Comment:
this is used to stop service as it is done previously when history fetching fails.

Reply:
done